config/src/main/resources/config-repo/auth-service.yml

```
@@ -77,4 +73,8 @@ eureka:
     service-url:
       defaultZone: http://${EUREKA_HOST:eureka-server:8761}/eureka
   instance:
-    prefer-ip-address: true
\ No newline at end of file
+    prefer-ip-address: true
+
+redis:
+  host: ${REDIS_HOST}
+  port: ${REDIS_PORT}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)